srcpkgs/libgit2/patches/0001-clar-remove-ftrunacte-from-libgit2-tests.patch

```diff
@@ -1,14 +1,5 @@
-From 12d73c418253a5c396465079c2808e07de17a1db Mon Sep 17 00:00:00 2001
-From: Peter Pettersson <boretrk@hotmail.com>
-Date: Thu, 14 Jul 2022 18:28:58 +0200
-Subject: [PATCH] clar: remove ftrunacte from libgit2 tests
-
----
- tests/libgit2/CMakeLists.txt | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
 diff --git a/tests/libgit2/CMakeLists.txt b/tests/libgit2/CMakeLists.txt
```

Comment:
```suggestion
Backported from:
https://github.com/libgit2/libgit2/commit/12d73c418253a5c396465079c2808e07de17a1db

diff --git a/tests/libgit2/CMakeLists.txt b/tests/libgit2/CMakeLists.txt
```

I see nothing wrong in removing the commit header, but it is better to indicate that you backported the patch from the "main" branch to clear confusion.

Reply:
Done